draft-packages/brand/KaizenDraft/Brand/Brand.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { Base } from "../../../illustration/KaizenDraft/Illustration/Base"
```

Comment:
Since each component is a package (generally) you'll want to import packages, not relative links. e.g.
```
import { Base } from "@kaizen/draft-illustration";
```

Though I wonder how useful this illustration abstraction is for this package, over using `assetUrl` 🤔

Reply:
Yeah I was unsure if it was overkill. Maybe I strip it out to keep it minimal and less dependant?